SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -637,7 +640,27 @@ CEulerSolver::CEulerSolver(CGeometry *geometry, CConfig *config, unsigned short
     Bleed_Area[iMarker]          = 0.0;
   }
   
-  /*--- Initialize quantities for the mixing process ---*/
+  /*--- Initializate quantities for SlidingMesh Interface ---*/
+  
+  SlidingState = new su2double** [nMarker];
+  
+  for (iMarker = 0; iMarker < nMarker; iMarker++){
+	  
+		if (config->GetMarker_All_KindBC(iMarker) == FLUID_INTERFACE){
+			
+			SlidingState[iMarker] = new su2double* [geometry->GetnVertex(iMarker)];
+			
+			for (iPoint = 0; iPoint < geometry->nVertex[iMarker]; iPoint++){
+					SlidingState[iMarker][iPoint] = new su2double[nPrimVar];
+					for (iVar = 0; iVar < nVar; iVar++)
+						SlidingState[iMarker][iPoint][iVar] = -1;
+			}
+		}
+		else
+			SlidingState[iMarker] = NULL;
+  }
+ 
```

Comment:
You're completely right thanks for pointing this out, Antonio. I'm planning to extend sliding meshes to NS in a future pull request.